### PR TITLE
Create what-why-how.md

### PR DESCRIPTION
**Pull request by jlovejoy**
update to text explaining SPDX License List (to be used in separate page or on top of license list page?)

### CODE REVIEW

**Comment by jlovejoy** [comment]
this is a new general description for the SPDX License List to potentially replace the first paragraph used on https://spdx.org/licenses/ and elsewhere.
adding it here as it's own page and then will work on how/where else to incorporate going forward

File: DOCS/what-why-how.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# About the SPDX License List
+
+## What
+The SPDX License List includes a list of commonly found free and open source licenses and exceptions, matching guidelines, and license expression language.  Such licenses may be used with open source software, hardware, data, documentation or other content. Each entry on the SPDX License List includes a short identifier, full name, vetted license text including matching guidelines markup as appropriate, and a canonical permanent URL. 
+
+## Why
+The purpose of the SPDX License List is to enable efficient and reliable identification of such licenses and exceptions in an SPDX document, in source files or elsewhere. The SPDX short identifiers combined with the matching guidelines ensures that anyone can reliably know exactly what license text is being referred to for a given SPDX identifier.
+
+## How
+* For more about how SPDX license identifiers are used in an SPDX document, a software bill of materials, or other places that store license data, see <LINKS TO SPECIFIC SECTIONS OF SPEC).
```
Comment (waldyrious) [inline DOCS/what-why-how.md]: @jlovejoy looks like this was not completed before the merge — or was that intentional?

File: DOCS/what-why-how.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+## Why
+The purpose of the SPDX License List is to enable efficient and reliable identification of such licenses and exceptions in an SPDX document, in source files or elsewhere. The SPDX short identifiers combined with the matching guidelines ensures that anyone can reliably know exactly what license text is being referred to for a given SPDX identifier.
+
+## How
+* For more about how SPDX license identifiers are used in an SPDX document, a software bill of materials, or other places that store license data, see <LINKS TO SPECIFIC SECTIONS OF SPEC). 
+* For examples of use of SPDX license identifiers in source code, see https://spdx.org/ids. 
+Please note, a license not on the SPDX License List can be included in an SPDX document by using a License Ref and including the full license text as per the specification. If you think there is a license that should be on the SPDX License List, please follow the instructions here <LINK TO CONTRIBUTING>
```
Comment (waldyrious) [inline DOCS/what-why-how.md]: Same here.

File: DOCS/what-why-how.md (hunk 1 above)
Comment (jlovejoy) [reply DOCS/what-why-how.md]: oops, you are right, will fix shortly!

File: DOCS/what-why-how.md (hunk 2 above)
Comment (jlovejoy) [reply DOCS/what-why-how.md]: see #863

**Pull request merged**